docs/protocol/papers/protocol-roadmap.md

```diff
@@ -8,23 +8,23 @@ If you have feature requests or feedback, we want to hear from you. Join us on [
 
 ## Denial of service mitigation
 
-* Explicit limits on number and size of blockchain entities (size of the blocks, number of transactions etc).
-* Fine-tuned runtime cost limits for the control and issuance programs.
-* Improvements to consensus algorithm.
+* Fine-tuned and tested runtime costs for the virtual machine
```

Comment:
Unless the list has super-short items, I prefer sentences to end with a period. It's weird for the eye to not hit the period after reading a whole line.